src/system/ZAuthModule/Controller/AccountController.php

```diff
@@ -373,7 +375,7 @@ public function changePasswordAction(Request $request)
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
             $data = $form->getData();
-            $mapping->setPass(\UserUtil::getHashedPassword($data['pass']));
+            $mapping->setPass($this->container->get('zikula_zauth_module.api.password')->getHashedPassword($data['pass']));
```

Comment:
Sometimes you use `$this->get`, sometimes `$this->container->get`. Please use `$this->get` only in controllers for consistency.